### DISCUSSION TITLE
蓝灯闪退无法使用的暂时解决办法 3月29日蓝灯闪退

### DISCUSSION THREAD

**Discussion by symbolyu950920** [getlantern/lantern-forum-cn#314] [General]
删除，蓝灯安装路径下的global.yaml文件
C:\Users\Administrator.DESKTOP-GINF5UQ\AppData\Roaming\Lantern\
这样可以正常打开，但是是免费版本的，你再正常激活，你这次只要不关闭蓝灯就能使用
应该蓝灯的认证出问题了

**Comment by mehuba** [comment]
Mac OS怎么做？

**Reply by Alinezhao** [reply]
我也想知道Mac OS怎么办

**Comment by symbolfung** [comment]
我试了你的方法也不行啊，系统win10家庭版

**Comment by konaisland** [comment]
这个办法会改变端口号，只能是临时办法。。。。实在是无语了

**Comment by casperwzy** [comment]
这个方法可以暂时使用 我先试试 还行 是免费版的 最起码暂时没有闪退

**Comment by MaxMKMax** [comment]
@symbolyu950920 您好. 谢谢您这位大神提供解决方案. 请各位展示参考一下. 

我们会尽快修复好. 谢谢你们耐心的等待!

**Comment by LuRenJia3053822459** [comment]
Windows 10 家庭版 22H2，删除了但是只有图标，连蓝灯的本地网页都无法打开激活，这免费版连谷歌网页都打不开

**Reply by MaxMKMax** [reply]
@LuRenJia3053822459 您可以参考一下外面的帖子. 有人分享怎么弄. 也有人说修好了. 您更新最新版本了吗?

**Comment by MaxMKMax** [comment]
各位好! 这两个链接都可以下载. 这个版本 7.8.4 应该修复了闪退和别的问题了. 如果还有什么问题, 请再来反馈.
谢谢你们耐心的等待!

https://github.com/getlantern/lantern/blob/devel/README.zh.md

https://github.com/getlantern/lantern

**Reply by Alinezhao** [reply]
感谢技术团队的努力，新版本可以上了。目前使用过程中还是经常会连不上，期待越来越好

**Reply by MaxMKMax** [reply]
@Alinezhao 您客气了. 非常感谢您支持我们. 这句鼓励的话会给我们团队很大的动力! 我来转达给他们. 谢谢!
